openapi.yaml

```diff
@@ -10509,3 +10570,45 @@ components:
       - created
       - data
       type: object
+    GoalTemplateList:
+      description: List of goal templates.
+      type: array
+      items:
+        $ref: '#/components/schemas/GoalTemplate'
+      example:
+      - id: 42
+        name: Untitled Template 0
+        metric: pr-release-time
+      - id: 43
+        name: Untitled Template 1
+        metric: jira-raised
+    GoalTemplate:
+      description: A template to generate a goal.
+      type: object
+      additionalProperties: false
+      properties:
+        id:
+          description: Identifier of the template.
+          type: integer
+        name:
+          $ref: '#/components/schemas/GoalTemplateName'
+        metric:
+          # {% if False %}
+          allOf:
+# {% else %}{{ "\n" }}          oneOf:{% endif %}
+          - $ref: '#/components/schemas/PullRequestMetricID'
+          - $ref: '#/components/schemas/JIRAMetricID'
+          - $ref: '#/components/schemas/ReleaseMetricID'
+      required:
+      - id
+      - name
+      - metric
```

Comment:
Same here, alphabetical sort. I wonder if the check can be automated...

Reply:
Interesting, yaml I think does not require order while reading keys so it is not immediate.

Reply:
This is more like Black for Yaml